csharp/ReadJsonFile.cs

```diff
@@ -0,0 +1,27 @@
+// Read JSON file, transform and print to stdout
+using System;
+using System.IO;
+using System.Text.Json;
+
+public class ReadJsonFile{
+    public static void Main(string[] args){
+        string jsonFilePath = args[0];
+        string jsonContent = File.ReadAllText(jsonFilePath);
+
+        JsonDocument doc = JsonDocument.Parse(jsonContent);
+        JsonElement root = doc.RootElement;
+
+        foreach (JsonElement person in root.EnumerateArray()){
+            if (person.ValueKind == JsonValueKind.Object){
+                if (person.TryGetProperty("age", out JsonElement ageElement) && ageElement.ValueKind == JsonValueKind.Number &&
```

Comment:
Instead of using `TryGetProperty` and testing the `ValueKind`, we should do something like:

```csharp
long age = person.GetProperty("age").GetInt64();
string firstName = person.GetProperty("first_name").GetString();
```